cmd/harmony/config.go

```diff
@@ -27,6 +27,7 @@ type harmonyConfig struct {
 	TxPool    txPoolConfig
 	Pprof     pprofConfig
 	Log       logConfig
+	Sys       sysConfig
```

Comment:
If we are to use make sysConfig hidden, simply make a pointer and add toml tag here.